packages/js/product-editor/src/components/attribute-control/new-attribute-modal.scss

```diff
@@ -90,14 +90,16 @@
 			position: absolute;
 			top: 36px;
 			z-index: 0;
-			display: none;
 		}
```

Comment:
We should remove all of the above styles, and that way we can also remove the styles added below.

Reply:
No, we can't. There is an issue in the core that renders an empty element even when the `label` property is `null.` We can remove it once [these PR changes](https://github.com/WordPress/gutenberg/pull/61336) are available in WordPress core.

Reply:
Hmm is this also still relevant now that we are always showing it?

Reply:
<img width="674" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/db38422e-296f-4a6d-a44f-fcc7c1606be7">

If we remove those lines, the help moves down, something that we want?